src/antsibull/build_ansible_commands.py

```diff
@@ -194,6 +194,30 @@ def make_dist(ansible_dir: str, dest_dir: str) -> None:
     shutil.move(os.path.join(dist_dir, files[0]), dest_dir)
 
 
+def make_dist_with_wheels(ansible_dir: str, dest_dir: str) -> None:
+    # pyre-ignore[16]
+    sh.python('setup.py', 'sdist', 'bdist_wheel', _cwd=ansible_dir)  # pylint:disable=no-member
+    dist_dir = os.path.join(ansible_dir, 'dist')
+    files = os.listdir(dist_dir)
+    tarball_count = 0
+    wheel_count = 0
+    for file in files:
+        if file.endswith('.tar') or file.endswith('.tar.gz'):
+            tarball_count += 1
+        elif file.endswith('.whl'):
+            wheel_count += 1
+        else:
+            tarball_count = 2  # the number is wrong, but this triggers an error
+            break
+    if tarball_count != 1 or wheel_count == 0:
+        raise Exception(
+            "python setup.py sdist bdist_wheel should have created exactly one tarball and at"
+            f" least one wheel (got {files})")
+
+    for file in files:
+        shutil.move(os.path.join(dist_dir, file), dest_dir)
```

Comment:
This looks good to me but also makes me wonder where the boundaries between python and ansible are -- as in, whether this could have been written as an [ansible task](https://opendev.org/zuul/zuul-jobs/src/branch/master/roles/build-python-release/tasks/main.yaml#L1-L10) or in python -- but we can have that discussion another time.

Reply:
That's a good question. I think it should be part of antsibull-build and not the role, though, so that you can simply re-create the source tarball (and in the future, also the wheel) from the extracted source tarball while only having to install antsibull (but not ansible itself).